test/acceptance/application-test.js

```diff
@@ -34,6 +34,11 @@ describe('Acceptance | application', function() {
         fixturesPath: 'tests'
       });
     }).then(function() {
+      app.editPackageJSON(function(pkg) {
+        pkg.devDependencies['ember-cli-fastboot'] = process.env.npm_package_devDependencies_ember_cli_fastboot;
```

Comment:
Running the fastboot server is not required to trigger this bug. I just grabbed a random package to test the npm install. Since we have a fastboot regression anyways it gets us closer to figuring that out. `npm_package_devDependencies_ember_cli_fastboot` is a way to always be in sync with package.json, so greenkeeper updates will always make it into the tests.